core/sqf/sql/scripts/sqstart

```diff
@@ -245,6 +245,63 @@ function checkKerberos {
    return 0
 }
 
+function printExpectedUlimt {
+
+    echoLog "desired \"ulimit -a\" output: "
+    echoLog "========================================="
+    echoLog "core file size (blocks, -c) 1000000"
+    echoLog "data seg size (kbytes, -d) unlimited"
+    echoLog "scheduling priority (-e) 0"
+    echoLog "file size (blocks, -f) unlimited"
+    echoLog "pending signals (-i) 515196"
+    echoLog "max locked memory (kbytes, -l) 49595556"
+    echoLog "max memory size (kbytes, -m) unlimited"
+    echoLog "open files (-n) 32000"
+    echoLog "pipe size (512 bytes, -p) 8"
+    echoLog "POSIX message queues (bytes, -q) 819200"
+    echoLog "real-time priority (-r) 0"
+    echoLog "stack size (kbytes, -s) 10240"
+    echoLog "cpu time (seconds, -t) unlimited"
+    echoLog "max user processes (-u) 267263"
+    echoLog "virtual memory (kbytes, -v) unlimited"
+    echoLog "file locks (-x) unlimited"
+    echoLog "========================================="
+ 
+}
+
+function checkUlimit {
+  
+ #check ulimit -u
+ userProc=`cat /proc/sys/kernel/pid_max`
+ if [[ $userProc -gt 65535 ]] ; then
+    echoLog "WARNING: detect issue during startup, please make sure your ulimit setup correctly"
```

Comment:
Thanks @traflm for the change. This is an error condition.  Because you have a way to flag it as ERROR, can you please change this to ERROR.